### ISSUE TITLE
Missing dependency in search path error when package-specs not wrapped in a list

### ISSUE THREAD

**Issue by tmepple** [jaredly/reason-language-server#83]
Using latest release `beta-1.0.0-beta.6-2` .vsix installed on MacOS 10.13.6...

Made a minimal repro of an issue I was having in my project where errors were emitted on each .re file like `File "", line 1:  Error: Missing dependency ReasonReact in search path`.  I found https://github.com/jaredly/reason-language-server/issues/68 which was very similar but the fix should have already been included in beta.6-2.

Here is the minimal repro:  https://github.com/tmepple/my-react-app

Basically this is taking the vanilla react theme and splitting the source files into 2 directories.

After hours of trial and error I think the issue is caused by:
```
  "package-specs": {
    "module": "es6",
    "in-source": true
  },
```

If I wrap `package-specs` in a list the issue goes away.

```
  "package-specs": [{
    "module": "es6",
    "in-source": true
  }],
```

**Comment by jaredly** [comment]
ah wow that's an annoying bug 😓 sorry for hours of trial an error!
